monix/shared/src/main/scala/org/atnos/eff/addon/monix/TaskEffect.scala

```diff
@@ -60,10 +59,12 @@ object TaskCreation extends TaskCreation
 trait TaskInterpretation extends TaskTypes {
 
   private val monixTaskMonad: MonadError[Task, Throwable] =
-    monix.cats.monixToCatsMonadError(Task.typeClassInstances.monadError)
+    MonadError[Task, Throwable]
 
-  private val monixTaskApplicative : Applicative[Task] =
-    monixToCatsApplicative(Task.nondeterminism.applicative)
+  private val monixTaskApplicative : Applicative[Task] = {
+    import Task.nondeterminism
```

Comment:
Could this perhaps be reduced to `Task.nondeterminism`?

Reply:
If I understand correctly `Task.nondeterminism` returns an implicit `ApplicativeStrategy[Task]` which in turn selects the right instance for `Applicative[Task]`.